app/api/events.py

```diff
@@ -45,11 +46,19 @@ class Meta:
                                schema='SocialLinkSchema',
                                many=True,
                                type_='social_link')
+    tracks = Relationship(attribute='track',
+                          self_view='v1.event_tracks',
+                          self_view_kwargs={'id': '<id>'},
+                          related_view='v1.track_list',
+                          related_view_kwargs={'event_id': '<id>'},
+                          schema='TrackSchema',
+                          many=True,
+                          type_='track')
 
     sponsor = Relationship(attribute='sponsor',
                            self_view='v1.event_sponsor',
                            self_view_kwargs={'id': '<id>'},
-                           related_view='v1.sponsor_detail',
+                           related_view='v1.sponsor_list',
```

Comment:
I was getting an error because of this, this is a one to many relationship, thats why the change